src/parser/mod.rs

```diff
@@ -8452,6 +8458,31 @@ impl<'a> Parser<'a> {
         }
     }
 
+    /// Parse a literal unicode normalization clause
+    pub fn parse_unicode_is_normalized(
+        &mut self,
+    ) -> Result<(Option<NormalizationForm>, bool), ParserError> {
```

Comment:
Oh the return type becomes a bit surprising by returning a bool, since this is a public API would it make sense to have it return the expr instead `Expr::IsNormalized` ?

Reply:
Sure; updated it as `-> Result<Expr, ParserError>` 👍